src/core/shaderlib/project/project.glsl.js

```diff
@@ -48,6 +49,9 @@ float project_scale(float meters) {
 }
 
 vec2 project_scale(vec2 meters) {
```

Comment:
`project_scale` is documented for projecting offset to pixels. `vec2 project_scale(vec2 meters)` is exclusively used for `xy` offsets. In LNG_LAT mode, this offset is in degrees rather than meters.

Maybe `meters` is not a good name for the argument?

Reply:
Haven't done a deep enough review to suggest alternatives but some initial comments

> Maybe meters is not a good name for the argument?

This could be a quite big change compared to my current way of thinking. All distances are supposed to be provided as meters in geospatial modes. I have tried to clearly state this in the docs e.g. the article about coordinate systems. The scale for distances does not change with`coordinateSystem`, i.e. any line widths or distances are still provided in meters even when in LNGLAT mode.

If there is a need for dealing with degree deltas for normals, we may want to add a different method to the project package.

> projecting offset to pixels

The project to "pixels" part is a little bit less firm. Most pixel work is now done after projection to clip space, so we may not have to worry too much about fixing the scale of the intermediate coordinate system.